src/test/java/bio/terra/common/auth/AuthService.java

```diff
@@ -37,9 +36,9 @@
   // this should always match exactly what the UI requests, so our tests represent actual user
   // behavior:
   private List<String> userLoginScopes =
-      Arrays.asList("openid", "email", "profile", "https://www.googleapis.com/auth/cloud-platform");
+      List.of("openid", "email", "profile", "https://www.googleapis.com/auth/cloud-billing");
```

Comment:
whats the reason for this being `cloud-billing` and not `cloud-billing.readonly`?

Reply:
I may need help from DevOps to update the dumbledore user to allow that scope. It's very strange.  For what it's worth, testing manually works

Reply:
Ok, I reduced the scopes requested by the tests so we should be all set now